src/axom/mir/tests/mir_clipfield.cpp

```diff
@@ -299,22 +307,34 @@ std::vector<int> makeRandomArray(int n)
   {
     values[i] = static_cast<int>(axom::utilities::random_real(0., largestId));
   }
-  return permute(values);
+  return values;
+}
+std::vector<double> makeRandomDoubleArray(int n)
+{
+  std::vector<double> values;
+  values.resize(n);
+  for(int i = 0; i < n; i++)
+  {
+    values[i] = drand48();
```

Comment:
`drand48` isn't available on windows.
Please use     `axom::utilities::random_real` instead.

See: https://github.com/LLNL/axom/pull/1519/commits/3a9f4e8c348f33200c35f0e33288041578f41832

Reply:
Fixed it.